torch/jit/_monkey_config_jit.py

```diff
@@ -0,0 +1,43 @@
+import monkeytype  # type: ignore
+
+from typing import Optional, Dict, Iterable, List, Set
+from monkeytype.db.base import CallTraceStore
+from monkeytype.config import default_code_filter
+from monkeytype.tracing import CallTrace, CodeFilter
+from collections import defaultdict
+
+def get_qualified_name(func):
+    return func.__module__ + '.' + func.__qualname__
+
+class JitTypeTraceStore(CallTraceStore):
+    def __init__(self):
+        super().__init__()
+        # A dictionary keeping all collected CallTrace
+        # key is fully qualified name of called function
+        # value is list of all CallTrace
+        self.trace_records = defaultdict(list)
+
+    def add(self, traces: Iterable[CallTrace]):
+        for t in traces:
+            qualified_name = get_qualified_name(t.func)
+            self.trace_records[qualified_name].append(t)
+
+    def filter(
+        self,
+        module: str,
```

Comment:
As we discussed offline, I think this API is meant to filter by module's fully qualified name, not function's.